source/core.py

```diff
@@ -291,18 +291,21 @@ def handleScreenOrientationChange(self, lParam):
 			import winUser
 			# Resolution detection comes from an article found at https://msdn.microsoft.com/en-us/library/ms812142.aspx.
 			#The low word is the width and hiword is height.
-			if (winUser.LOWORD(lParam) > winUser.HIWORD(lParam):
-				if self.orientationCache == self.ORIENTATION_LANDSCAPE:
+			width = winUser.LOWORD(lParam)
+			height = winUser.HIWORD(lParam)
+			if width > height:
+				#Check to see if the height and width are the same. If so, it's actually a screen flip, and we want to alert of those!

```

Comment:
I feel like this comment could be a bit misleading, as it suggests you're checking whether they're the same (but you're actually checking whether they're different). Perhaps this:

`# If the height and width are the same, it's actually a screen flip, and we do want to alert of those!`